error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/FileRules.java

```diff
@@ -18,6 +21,49 @@
 final class FileRules {
   private FileRules() {}
 
+  /** Prefer the more idiomatic {@link Path#of(URI)} over {@link Paths#get(URI)}. */
+  static final class PathOfUri {
+    @BeforeTemplate
+    Path before(URI uri) {
+      return Paths.get(uri);
+    }
+
+    @AfterTemplate
+    Path after(URI uri) {
+      return Path.of(uri);
+    }
+  }
+
+  /**
+   * Prefer the more idiomatic {@link Path#of(String, String...)} over {@link Paths#get(String,
+   * String...)}.
+   */
+  static final class PathOfString {
+    @BeforeTemplate
+    Path before(String first, @Repeated String more) {
+      return Paths.get(first, more);
+    }
+
+    @AfterTemplate
+    Path after(String first, @Repeated String more) {
+      return Path.of(first, more);
+    }
+  }
```

Comment:
I recently added these two to my TODO list because I saw openrewrite/rewrite-migrate-java#519, but just now I noticed there was already an older identical TODO entry :smile:

Reply:
One of the rare cases where our solutions are equal in length. ;)